test/unit/rules/no-curly-component-invocation-test.js

```diff
@@ -275,13 +286,61 @@ generateRuleTests({
           message: generateError('foo.bar'),
           line: 1,
           column: 0,
+          isFixable: false,
           source: '{{foo.bar}}',
         },
       ],
     },
   ],
 });
 
+// fixers tests
+
+generateRuleTests({
```

Comment:
Can you add more tests to this list to ensure that custom whitespace (within a single hash argument) is preserved?

Reply:
assert for empty results without { template, fixedTemplate } added, tests added